examples/express/with-emailpassword/backend/package.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "supertokens-node",
+    "version": "0.0.1",
+    "private": true,
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "start": "npx ts-node-dev --project ./tsconfig.json ./index.ts"
+    },
+    "dependencies": {
+        "cors": "^2.8.5",
+        "express": "^4.18.1",
+        "helmet": "^5.1.0",
+        "morgan": "^1.10.0",
+        "npm-run-all": "^4.1.5",
+        "supertokens-node": "latest",
+        "ts-node-dev": "^2.0.0",
+        "typescript": "^4.7.2"
```

Comment:
this should be empty cause all these should be in the package.json of the with-emailpassword folder.